sci-electronics/systemc/systemc-2.3.4-r1.ebuild

```diff
@@ -41,12 +40,27 @@ src_configure() {
 		--with-unix-layout
 }
 
+src_install() {
+	default
+	if use doc; then
+		if use examples; then
+			docompress -x /usr/share/doc/"${PF}"/examples
+		else
+			rm -r "${ED}"/usr/share/doc/"${PF}"/examples
+		fi
+	else
+		rm -r "${ED}"/usr/share/doc/"${PF}"
+	fi
```

Comment:
use 'rm || die', since it can always fail

Reply:
> use 'rm || die', since it can always fail

Done, all command not from eclass have add ``|| die`` to catch fail